pom.xml

```diff
@@ -108,6 +108,11 @@
             <artifactId>maven-model</artifactId>
             <version>3.3.9</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
OK, so we should at least try to estimate if there would be any impact if we include it as is. Normally the dependencies will only clash if the lib is also in core or any plugins that this plugin depends on has it as well. If that's not the case then it should be safe to add as is.